app/src/main/java/ru/vssemikoz/newsfeed/models/Category.java

```diff
@@ -0,0 +1,20 @@
+package ru.vssemikoz.newsfeed.models;
+
+public enum Category {
+    all("Все"),
```

Comment:
1. Enums are constants. All constants must be named with the uppercase, i.e `ALL`.
2. Please, move all strings to strings.xml as they are resources and subjects to change.